tests/foreman/api/test_permission.py

```diff
@@ -31,48 +34,51 @@ class PermissionsTestCase(APITestCase):
     """Tests for the ``permissions`` path."""
 
     @run_only_on('sat')
-    @ddt_data(*PERMISSIONS_NAME)  # pylint:disable=W0142
+    @ddt_data(*PERMISSION_NAMES)  # pylint:disable=W0142
     def test_search_by_name(self, permission_name):
-        """@test: permissions can be searched by name
+        """@test: Search for a permission by name.
 
         @feature: Permissions
 
-        @assert: Searched permission name match the query name
+        @assert: Only one permission is returned, and the permission returned
+        is the one searched for.
 
         """
         result = entities.Permission(name=permission_name).search()
-        self.assertEqual(len(result), 1)
+        self.assertEqual(len(result), 1, permission_name)
         self.assertEqual(permission_name, result[0]['name'])
 
     @run_only_on('sat')
-    @ddt_data(*PERMISSIONS_RESOURCE_TYPE)  # pylint:disable=W0142
+    @ddt_data(*PERMISSION_RESOURCE_TYPES)  # pylint:disable=W0142
     def test_search_by_resource_type(self, resource_type):
-        """@test: permissions can be searched by resource type
+        """@test: Search for permissions by resource type.
 
-        @feature: Permissions - Search
+        @feature: Permissions
```

Comment:
All of the tests in class `PermissionsTestCase` call method `Permission.search`, so I see little need for " - Search".